UDPBroadcast/UDPBroadcastConnection.swift

```diff
@@ -189,27 +189,30 @@ open class UDPBroadcastConnection {
             
             guard sent > 0 else {
                 if let errorString = String(validatingUTF8: strerror(errno)) {
-                    print("UDP connection failed to send data: \(errorString)")
+                    debugPrint("UDP connection failed to send data: \(errorString)")
                 }
                 closeConnection()
                 return
             }
             
             if sent == broadcastMessageLength {
                 // Success
-                print("UDP connection sent \(broadcastMessageLength) bytes")
+                debugPrint("UDP connection sent \(broadcastMessageLength) bytes")
             }
         }
     }
     
     /**
      Close the connection.
      */
-    open func closeConnection() {
+    open func closeConnection(_ reopen: Bool = true) {
```

Comment:
It seems that you forgot to implement the reopen functionality. Furthermore I don't like the use of an anonymous parameter ("_") here. Users of this function can't guess what a call like `closeConnection(true)` means because it would rather suggest that the call closes the connection for good, forced or similar.

Reply:
oops...

Reply:
closing this because outdated after my last commit